community/examples/slurm-gcp-v5-high-io.yaml

```diff
@@ -26,7 +26,7 @@ vars:
   disable_public_ips: false
   # Set to true for active cluster reconfiguration. Note that setting this
   # option requires additional dependencies to be installed locally.
-  enable_reconfigure: false
+  enable_reconfigure: true
```

Comment:
It is not obvious to me why we want this on in this example. We do not specify it in other v5 examples and the default is false. Is there something special about this example? Do we just want test coverage of this option?

I see good documentation for the required dependencies for this option but am not sure how the errors actually manafest and so am wondering if this is something we want in a public example.

Reply:
The high IO example in general has been more "full featured", so I wanted to include this since it's one of the most impactful new features in v5. 

Any errors will be clear in the terraform output - they are needed for a "null resource" script that's run locally, and the error specifies clearly which package is needed.

There is also the benefit of having it included in our tests, though I'd rather find a better way of adding additional command line variables in tests then set it by default for everyone if that's the only reason.